### PR TITLE
TLF-42-pipeline: Add the new slack channels and secrets

### PR DESCRIPTION
**Pull request by adityababumallisettiHO**
## What? 
All the HOF services have the slack notifications but they are misconfigured. sas-build channel is archived. We have decided to use exisiting slack channel to notify us  build status of security scanners and tear down envs
Cron jobs are useful for monitoring the execution and status of the cron jobs and help in quickly identifying issues, ensuring that the jobs are running as expected

## Why? 
These notifications are important and alert us for any vulnerabilities detected in cron security scanners
We will also be notified about cron steps build success and failures tearing down envs created by pull requests save  on infrastructure costs

## How? 
I have added the webhooks as secrets to the Drone secrets. Updated the Slack channels for the two types of cron jobs.

## Testing?
We will monitor them after merging to Master. I have tested by sending curl request to post to a channel for both the slack channels.

## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -425,14 +425,14 @@ steps:
     pull: if-not-exists
     image: plugins/slack
     settings:
-      channel: sas-build
+      channel: sas-hof-build-notify
       failure: ignore
       icon_url: http://readme.drone.io/0.5/logo_dark.svg
```
Comment (dk4g) [inline .drone.yml:430]: that url returns 404.
as agreed to use this drone icon instead
`https://readme.drone.io/logo.svg`

Comment (dk4g) [reply .drone.yml:430]: icon.url appears twice

File: .drone.yml [hunk 2]
```diff
@@ -425,14 +425,14 @@ steps:
...
       icon.url: http://readme.drone.io/0.5/logo_dark.svg
       template: "CRON Job {{build.deployTo}} of Change of Address has {{build.status}} - <{{build.link}}|#{{build.number}}> {{#success build.status}}\n  :thumbsup: :thumbsup: :thumbsup:\n{{else}}\n  :x: :x: :x:\n{{/success}} Author: {{build.author}}\n\nDuration: {{since job.started}}\n\nJob: <{{build.link}}|#{{build.number}}>\n\nCommit: {{build.commit}}\n"
```
Comment (dk4g) [inline .drone.yml:432]:
```
template: >
            :x: CRON Job `tear_down_pr_envs` for Change of Address (COA) has failed.
            Repo `{{ repo.name }}` branch <https://github.com/{{ repo.owner }}/{{ repo.name }}/commits/{{ build.branch }}|{{build.branch}}> for commit <https://github.com/{{ repo.owner }}/{{ repo.name }}/commit/{{ build.commit }}|{{ truncate build.commit 8 }}>
```

File: .drone.yml [hunk 3]
```diff
@@ -449,7 +449,7 @@ steps:
...
```
Comment (dk4g) [inline .drone.yml:449]:
```
template: >
            :x: CRON Job `security_scans` for Change of Address (COA) has failed.
            Repo `{{ repo.name }}` branch <https://github.com/{{ repo.owner }}/{{ repo.name }}/commits/{{ build.branch }}|{{build.branch}}> for commit <https://github.com/{{ repo.owner }}/{{ repo.name }}/commit/{{ build.commit }}|{{ truncate build.commit 8 }}>
```

File: .drone.yml (hunk 1 above)
Comment (adityababumallisettiHO) [reply .drone.yml]: Thanks @dk4g . I have corrected the image url

File: .drone.yml (hunk 2 above)
Comment (adityababumallisettiHO) [reply .drone.yml]: Thank you very much @dk4g . Much appreciated . I have updated in the code

File: .drone.yml [hunk 4]
```diff
@@ -425,14 +425,14 @@ steps:
     pull: if-not-exists
     image: plugins/slack
     settings:
-      channel: sas-build
+      channel: sas-hof-build-notify
       failure: ignore
       icon_url: http://readme.drone.io/0.5/logo_dark.svg
       icon.url: http://readme.drone.io/0.5/logo_dark.svg
       template: "CRON Job {{build.deployTo}} of Change of Address has {{build.status}} - <{{build.link}}|#{{build.number}}> {{#success build.status}}\n  :thumbsup: :thumbsup: :thumbsup:\n{{else}}\n  :x: :x: :x:\n{{/success}} Author: {{build.author}}\n\nDuration: {{since job.started}}\n\nJob: <{{build.link}}|#{{build.number}}>\n\nCommit: {{build.commit}}\n"
       username: Drone
```
[the remark this answers is not in the document]
Comment (adityababumallisettiHO) [reply .drone.yml]: I have used Drone_CI

**Review by dk4g — APPROVED** [review]

**Pull request merged**